browser/ui/webui/settings/brave_wallet_handler.cc

```diff
@@ -135,12 +120,72 @@ void BraveWalletHandler::GetCustomNetworksList(
   ResolveJavascriptCallback(args[0], base::Value(json_string));
 }
 
+bool BraveWalletHandler::AddEthereumChainToRPCService(
+    base::Value javascript_callback,
+    const std::string& payload,
+    std::string* error_message) {
+  auto value_to_add = GetEthereumChain(payload, error_message);
+  if (!value_to_add) {
+    *error_message = l10n_util::GetStringUTF8(
+        IDS_SETTINGS_WALLET_NETWORKS_SUMBISSION_FAILED);
```

Comment:
Isn't this unnecessary as it should have already been set in https://github.com/brave/brave-core/pull/11971/files#diff-4485cfb12a4c60f887f9e1929fc5c346c50422ee430645a68334cb360c97989fR43-R45?

Reply:
done